lesson24/features/pageobjects/Base.ts

```diff
@@ -0,0 +1,19 @@
+export class Base {
+
+    async isContainingInUrl(text: string) {
```

Comment:
this method can only return true, I do not this that this method make sense in this view